bip-0002.mediawiki

```diff
@@ -54,6 +54,7 @@ For a BIP to be accepted it must meet certain minimum criteria.
 It must be a clear and complete description of the proposed enhancement.
 The enhancement must represent a net improvement.
 The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+BIP drafts should not be merged until the idea has materially progressed beyond the ideation phase - it should have multiple implementations from independent authors released in different software packages, and/or have received material public discussion feedback from diverse contributors with the BIP champion actively working towards adoption for an extended time period. While BIPs are author documents which must only meet certain minimum criteria, the BIP editors should strive to ensure there are not an unnecessary number of BIPs which never progress into broad implementation across the bitcoin ecosystem.
```

Comment:
This sounds like a fine idea, but I’m not convinced by the approach. So far, implementation was considered a requirement for a BIP to move to _Final_, and the _Draft_ status was used for a BIP that was not yet recommended to be implemented, or more generally still open to major changes. Under BIP2, number assignment had been happening when BIPs got merged, so in _Draft_ status.

While I agree that it’s not useful to have a huge number of BIPs that never get adopted, it already takes significant effort to write up ideas sufficiently comprehensive to meet the "high quality" and "provides sufficient information to implement from the BIP" criteria.
If these two sentences were incorporated into our process, it would mean that we would only assign numbers to BIPs that are ready to progress to _Final_. This is a complete overhaul of the prior process and as such I would concur that this amendment rather be taken into account for a successor Process BIP instead of a change to BIP2.
Either way, it seems to me that a number assignment would be useful long before a BIP is ready to progress to _Final_. 

Perhaps when we update our process, we could consider an approach where a BIP can be merged to the repository as a _Draft_ before assigning a number, named _"BIP-draft-author-topic"_, and then a number be assigned when a BIP progresses to _Proposed_.